### PR TITLE
ChildrenScreen: navigation $ icon

### PR DESCRIPTION
**Pull request by HyoukSunKwon**

### CODE REVIEW

File: src/screens/ChildrenScreen.js [hunk 1]
```diff
@@ -1,158 +1,126 @@
...
 
-import colors from "../config/colors";
+import { AuthContext } from "../config/context";
+import { scale, moderateScale, verticalScale } from "../config/scaling";
 import screen from "../config/screen";
+import colors from "../config/colors";
+import { TouchableOpacity } from "react-native-gesture-handler";
+import { Card } from "react-native-elements";
+import { Ionicons } from "@expo/vector-icons";
+import AddChildScreen from "./AddChildScreen";
```
Comment (hyunjiLeeTech) [inline src/screens/ChildrenScreen.js:11]: Erase the modules you don't use on this screen. One of the modules you don't use creates an error.
image

File: src/screens/ChildrenScreen.js [hunk 2]
```diff
@@ -1,158 +1,126 @@
...
     flex: 1,
-    position: "absolute",
+  },
+  addIcon: {
     alignSelf: "flex-end",
-    bottom: 0,
-    right: 0,
+    width: moderateScale(70),
+    height: moderateScale(70),
+    marginRight: moderateScale(23),
```
Comment (hyunjiLeeTech) [inline src/screens/ChildrenScreen.js:123]: Use scale for width size/marginRight/marginLeft and verticalScale for height size/marginTop/marginbottom. Please check this and fix it on the other screens you created (EditChildrenScreen)

**Review by ElisaNgLi — COMMENTED** [review]
For the Children Screen remember to add the Icons for the Birth and Latest like is displayed on the design 
Screen Shot 2021-05-14 at 5 15 26 PM

File: src/screens/ChildrenScreen.js [hunk 3]
```diff
@@ -1,158 +1,137 @@
...
-                </TouchableOpacity>
-                <Ionicons
-                  name="ios-arrow-forward"
-                  size={30}
-                  color={colors.lightBlue}
-                  style={styles.arrowIcon}
-                ></Ionicons>
+            <View key={u.id}>
+              <View style={styles.container}>
+                {/* <Image source={{ uri:u.avatar }} /> */}
```
Comment (hyunjiLeeTech) [inline src/screens/ChildrenScreen.js:40]: Please use TouchableOpacity and make it to navigate to ScheduleScreen when user click a item.

File: src/screens/ChildrenScreen.js [hunk 4]
```diff
@@ -1,158 +1,137 @@
...
-    position: "absolute",
     alignSelf: "flex-end",
-    bottom: 0,
-    right: 0,
+    width: moderateScale(70),
+    height: moderateScale(70),
+    marginRight: moderateScale(23),
+  },
+  icon: {
+    width: moderateScale(10),
```
Comment (hyunjiLeeTech) [inline src/screens/ChildrenScreen.js:132]: Add 
resizeMode: "stretch"
You can see code in the file ScheduleScreen as a reference

File: src/screens/ChildrenScreen.js [hunk 5]
```diff
@@ -1,158 +1,137 @@
...
-    width: "95%",
-    marginLeft: "5%",
-    marginTop: "2%",
-    paddingBottom: "2%",
-    borderBottomColor: colors.lightBlue_button,
-    borderBottomWidth: 1,
+    height: moderateScale(96),
+    backgroundColor: colors.white,
+    flexDirection: "row",
+    marginBottom: scale(10),
```
Comment (hyunjiLeeTech) [inline src/screens/ChildrenScreen.js:93]: Use scale for width and verticalScale for height (please recheck whole your stylesheet codes). You can recheck the logic of scaling scaling.js file

File: src/screens/ChildrenScreen.js [hunk 6]
```diff
@@ -1,158 +1,137 @@
...
+    flexDirection: "row",
+  },
+  name: {
+    fontFamily: "notoSans-regular",
+    fontSize: moderateScale(16),
     fontWeight: "bold",
-    color: colors.lightBlue,
+    lineHeight: moderateScale(22),
+    margin: scale(5),
   },
```
Comment (hyunjiLeeTech) [inline src/screens/ChildrenScreen.js:104]: The start point of name and the icons (schedule and infection icons) should be the same. Currently, it doesn't like the same.

File: src/screens/ChildrenScreen.js [hunk 7]
```diff
@@ -1,158 +1,137 @@
...
-  date: {
-    fontSize: 15,
+  birth: {
+    fontFamily: "notoSans-regular",
+    fontSize: moderateScale(12),
+    color: "#1F263E",
+    lineHeight: moderateScale(16.34),
+    margin: scale(5),
```
Comment (hyunjiLeeTech) [inline src/screens/ChildrenScreen.js:110]: Please reduce the distance a little bit so the start end point of the image (profile image) and the end point of the texts (latest) is the same. (In other words, the height of image and the height of name+birth+latest should be the same)

**Review by hyunjiLeeTech — CHANGES_REQUESTED** [review]

**Pull request closed without merging**